packages/poi-preset-karma/index.js

```diff
@@ -57,7 +57,11 @@ module.exports = (options = {}) => {
           }]
         },
         preprocessors: files.reduce((current, next) => {
-          current[next] = ['webpack', 'sourcemap']
+          if (typeof next === 'object' && Boolean(next.included)) {
```

Comment:
what if `included` is absent here? it should be included as well since it defaults to `true`

Reply:
If absent it will not be included. Let me look up the default value. Thought it to be true.

Reply:
Yeah the default value is true, so it should be good.

```
included
Type. Boolean
Default. true
```

[docs](http://karma-runner.github.io/1.0/config/files.html)

Reply:
if it's absent `Boolean(next.included)` will be false 😅

Reply:
Doh, sorry -- will correct that (and have some coffee)

Reply:
hah we can simply include it by checking `next.included !== false` I guess

Reply:
Yes, that is correct

Reply:
I don't think people would use other truthy or falsy values than true/false.